File: ex02/ex02_05/ex02_05.py
def main():
	  cmd = input("Enter string: ")
	  print("First character is alphabetic: " + str(cmd[0].isalpha()))
	  new_cmd = cmd[0].upper()
	  new_cmd += cmd[1:len(cmd)]
	  print(new_cmd)
	  return 0

if __name__ == "__main__":
  	main()

File: ex02/ex02_05/ex02_05.js
import promptSync from 'prompt-sync';
const prompt = promptSync();

function main() {
  	let cmd = "";
  	let new_cmd = "";
  	let regExp = /[A-Za-z]/;
  	cmd = prompt("Enter string: ");
  	console.log("First character is alphabetic: " + regExp.test(cmd[0]));
  	new_cmd = cmd[0].toUpperCase();
  	new_cmd += cmd.slice(1, cmd.length);
  	console.log(new_cmd);
  	return 0;
}
main();